edenia/eden-spend-explorer
# feat(webapp): implement new homepage design

### Implement new homepage design

### What does this PR do?

- Resolve #335 

### Steps to test

1. Run the project and go to the home view and see the new change

![image](https://user-images.githubusercontent.com/56526123/208997699-155416c1-e1b6-4c76-bab2-905321381d2f.png)
